Source/SwiftLintFramework/Rules/Lint/EmptyXCTestMethodRule.swift

```diff
@@ -15,15 +15,29 @@ public struct EmptyXCTestMethodRule: OptInRule, ConfigurationProviderRule, Swift
     )
 
     public func makeVisitor(file: SwiftLintFile) -> ViolationsSyntaxVisitor? {
-        EmptyXCTestMethodRuleVisitor(viewMode: .sourceAccurate)
+        EmptyXCTestMethodRuleVisitor(testParentClasses: configuration.testParentClasses)
+    }
+    
+    public func makeViolation(file: SwiftLintFile, position: AbsolutePosition) -> StyleViolation {
+        StyleViolation(
+            ruleDescription: Self.description,
+            severity: configuration.severity,
+            location: Location(file: file, position: position)
+        )
     }
 }
 
 private final class EmptyXCTestMethodRuleVisitor: SyntaxVisitor, ViolationsSyntaxVisitor {
     private(set) var violationPositions: [AbsolutePosition] = []
+    private let testParentClasses: Set<String>
 
+    init(testParentClasses: Set<String>) {
+        self.testParentClasses = testParentClasses
+        super.init(viewMode: .sourceAccurate)
```

Comment:
Please initialize the base class first before applying your own customizations:
```suggestion
        super.init(viewMode: .sourceAccurate)
        self.testParentClasses = testParentClasses
```

Reply:
Flipping them around gives me `Property 'self.testParentClasses' not initialized at super.init call` and `Immutable value 'self.testParentClasses' may only be initialized once` :-(

Reply:
You are right. In Swift, the subclass has to initialize their properties first. Confused it with other languages.